docs/content/docs/tutorials/apigatewayv2-reference-example.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "Manage HTTP API using the ACK APIGatewayv2 Controller"
+description: "Create and Invoke an Amazon APIGateway HTTP API using ACK APIGatewayv2 controller deployed on Amazon Elastic Kubernetes Service (EKS)."
+lead: "Create and invoke an Amazon APIGateway HTTP API using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 43
+toc: true
+---
+
+The ACK service controller for Amazon APIGatewayv2 lets you manage HTTP APIs and VPC Links directly from Kubernetes.
+This guide will show you how to create and invoke an HTTP API using a single Kubernetes resource manifest.
```

Comment:
I think this tutorial does a good job of showing *what* is happening, but not *why*. I understand it's not meant to be a tutorial for APIGWv2, but could you add a little bit more information about what this code actually deploys? What is the end result of this tutorial, how it could be used as the basis for something else, etc. If this is using an example setup from the APIGWv2 documentation, or another repo, it would be good to link to those docs too.

Reply:
Added more context : )